.github/workflows/rust.yml

```
@@ -2,132 +2,104 @@
-        timeout-minutes: 5 # 300 seconds
-        # Use cargo nextest run instead of cargo test
-        run: cargo nextest run --all-features 
-
-  coverage:
-    name: cargo tarpaulin
-    runs-on: ubuntu-latest
-    needs:
-      - unit-tests
-      - clippy
-      - rustfmt
+    - uses: actions/checkout@v4
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)